ractive-koa/server/services/routes.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// var config = require('./config');
+
+module.exports = function(ss, app, router) {
+
+	// x-response-time
+
+	app.use(function*(next) {
+		var start = new Date(), ms;
+		yield next;
+		ms = new Date() - start;
+		this.set('X-Response-Time', ms + 'ms');
+	});
+
+	// logger
+
+	app.use(function*(next) {
+		var start = new Date(), ms;
+		yield next;
+		ms = new Date() - start;
+		console.log('%s %s - %s', this.method, this.url, ms);
+	});
+
+	// Define a single-page client called 'main'
+	ss.client.define('main', {
+		view: 'app.jade',
+		css: ['app.styl'],
+		code: ['libs/jquery.min.js', 'app'],
+		tmpl: '*'
+	});
+
+	// response
+
+	// // Serve this client on the root URL
+	// ss.http.route('/', function(req, res) {
+	// 	console.log('getting this far in the world');
+	// 	res.serveClient('main');
+	// });
+
+	router.get('/', function*(next) {
```

Comment:
With middleware mounted, can just use Koa to serve the client. But can also use `ss.http.route('/', ...` which I've commented out above, for reference. Either works.